src/content/docs/en/reference/routing-reference.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Routing Reference
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 4
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+There is no separate routing configuration in Astro. Every [supported page file](/en/basics/astro-pages/#supported-page-files) located within the special `src/pages/` directory creates a single page route, or multiple page routes dynamically when the file name contains a [parameter](#params).
+
+
+## `getStaticPaths()`
+
+**Type:** `(options: GetStaticPathsOptions) => Promise<GetStaticPathsResult> | GetStaticPathsResult`
+
+If a page uses dynamic params in the filename, that component will need to export a `getStaticPaths()` function.
+
+This function is required because Astro is a static site builder. That means that your entire site is built ahead of time. If Astro doesn't know to generate a page at build time, your users won't see it when they visit your site.
```

Comment:
```suggestion
This function is required because Astro is a static site builder. That means that your entire site is built ahead of time. If Astro doesn't know how to generate a page at build time, your users won't see it when they visit your site.
```